<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>client</class>
 <widget class="QMainWindow" name="client">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>661</width>
      <height>451</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>971</width>
      <height>601</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(:/QT/icon/backfin.png);
border-radius: 20px;

</string>
    </property>
    <widget class="QToolButton" name="pwd">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>230</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="Client.qrc">
       <normaloff>:/Client/QT/icon/Password.png</normaloff>:/Client/QT/icon/Password.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>171</width>
       <height>80</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 italic 28pt &quot;Comic Sans MS&quot;;
</string>
     </property>
     <property name="text">
      <string>Welcome</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>410</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>注册</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>180</y>
       <width>191</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent;border-width:0;border-style:outset</string>
     </property>
     <property name="text">
      <string>————————————————</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>260</y>
       <width>191</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent;border-width:0;border-style:outset</string>
     </property>
     <property name="text">
      <string>————————————————</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_pwd">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>240</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>密码：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_username">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>160</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>账号：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_pwd">
     <property name="geometry">
      <rect>
       <x>225</x>
       <y>220</y>
       <width>191</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
background:transparent;border-width:0;border-style:outset</string>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_username">
     <property name="geometry">
      <rect>
       <x>225</x>
       <y>150</y>
       <width>191</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent;border-width:0;border-style:outset;
font: 75 14pt &quot;Comic Sans MS&quot;;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>10</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(192, 191, 188);</string>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QToolButton" name="pushButton_login">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>290</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style:none;
font: 28pt &quot;华文隶书&quot;;
background-color: transparent;
border-width:0;</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="Client.qrc">
       <normaloff>:/Client/QT/icon/Login.png</normaloff>:/Client/QT/icon/Login.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
    </widget>
    <widget class="QToolButton" name="pushButton_register">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>360</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style:none;
font: 10pt &quot;华文隶书&quot;;</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="Client.qrc">
       <normaloff>:/Client/QT/icon/Register.png</normaloff>:/Client/QT/icon/Register.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>90</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style:none;
image:url(:/Client/QT/icon/Chat.png)</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_2">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>160</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="Client.qrc">
       <normaloff>:/Client/QT/icon/login_Account.png</normaloff>:/Client/QT/icon/login_Account.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="Client.qrc"/>
 </resources>
 <connections/>
</ui>
